docs/source/conf.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+#
+# IPython.parallel documentation build configuration file, created by
+# sphinx-quickstart on Mon Apr 13 12:53:58 2015.
+
+# NOTE: This file has been edited manually from the auto-generated one from
+# sphinx.  Do NOT delete and re-generate.  If any changes from sphinx are
+# needed, generate a scratch one and merge by hand any new fields needed.
+
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+import shlex
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#sys.path.insert(0, os.path.abspath('.'))
+
+# We load the ipython release info into a dict by explicit execution
+iprelease = {}
+exec(compile(open('../../ipython_parallel/_version.py').read(), '../../ipython_parallel/_version.py', 'exec'),iprelease)
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
+    'sphinx.ext.doctest',
+    'sphinx.ext.inheritance_diagram',
+    'sphinx.ext.intersphinx',
+    'numpydoc',  # to preprocess docstrings
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# Add dev disclaimer.
+if iprelease['_version_extra'] == 'dev':
+    rst_prolog = """
+    .. note::
+
+        This documentation is for a development version of IPython. There may be
+        significant differences from the latest stable release.
+
+    """
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'IPython.parallel'
+copyright = u'2015, The IPython Development Team'
+author = u'The IPython Development Team'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = iprelease['version'].split('-', 1)[0]
+# The full version, including alpha/beta/rc tags.
+release = iprelease['version']
```

Comment:
From the sphinx docs, I think `release` should be `__version__`, and `version` should be `'.'.join(map(str, version_info[:2]))`